pkg/database/postgres_db_instance.go

```
@@ -0,0 +1,83 @@
+package database
+
+import (
+	"fmt"
+	glog "log"
+	"net/url"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
+	"gorm.io/plugin/dbresolver"
+)
+
+// PostgresDbInstance is the Postgres-specific DbInstance variant
+type PostgresDbInstance struct {
+	DbInstance
+}
+
+// Reset implementation for this type
+func (pgdb PostgresDbInstance) Reset() error {
+	log.Info("Resetting database schema")
+	pgdb.Db().Exec("drop schema public cascade")
+	pgdb.Db().Exec("create schema public")
```

exec should return an error? so we have to process it.